rust/src/measurements/gaussian/mod.rs

```diff
@@ -79,13 +79,12 @@ where
 /// If k is not set, k defaults to the smallest granularity.
 ///
 /// # Arguments
-/// * `input_domain` - Domain of the data type to be privatized.
-/// * `input_metric` - Metric of the data type to be privatized.
+/// * `input_domain` - Domain of the data type to be privatized. Valid values are `VectorDomain<AtomDomain<T>>` or `AtomDomain<T>`.
+/// * `input_metric` - Metric of the data type to be privatized. Valid values are `AbsoluteDistance<T>` or `L2Distance<T>`.
```

Comment:
I thought you were switching these to use constructor form (i.e. `atom_domain(T)`)?